packages/webapp/src/utils/developer-constants.js

```diff
@@ -88,6 +90,9 @@ const DEVELOPER_PLANNING_DECISION_FILE_SIZE = 'planning-decision-file-siz'
 const DEVELOPER_PLANNING_DECISION_FILE_TYPE = 'planning-decision-file-type'
 const DEVELOPER_PLANNING_DECISION_FILE_NAME = 'planning-decision-file-name'
 const DEVELOPER_PLANNING_DECISION_CHECKED = 'planning-decision-checked'
+const DEVELOPER_PLANNING_AUTHORITY_LIST = 'planning-authority-list'
+const DEVELOPER_PLANNING_APPLICATION_REF = 'planning-application-ref'
+const DEVELOPER_DEVELOPMENT_NAME = 'planning-development-name'
```

Comment:
These three keys should have `developer-` prefixed to them (e.g. `developer-planning-development-name`) to avoid clashes with registration journey keys

Reply:
done